doc/DoxygenLayout.xml

```diff
@@ -5,7 +5,11 @@
     <tab type="mainpage" visible="yes" title=""/>
     <tab type="user" visible="yes" title="Tutorial" url="https://github.com/ginkgo-project/ginkgo/wiki/Tutorial:-Building-a-Poisson-Solver" />
     <tab type="user" visible="yes" title="Examples" url="@ref Examples" />
-    <tab type="user" visible="yes" title="Citing Ginkgo" url="@ref citing_ginkgo" />
+    <tab type="user" visible="yes" title="Publications" url="@ref citing_ginkgo" >
+      <tab type="user" visible="yes" title="Citing Ginkgo" url="@ref citing_ginkgo" />
+      <tab type="user" visible="yes" title="Ginkgo White Paper" url="https://www.arxiv-vanity.com/papers/2006.16852/" />
```

Comment:
I looked at the formatting, and it seems like they have some issues with code formatting (they center-align the code and have fixed line numbers that don't move when horizontal-scrolling), so maybe the arxiv URL might be better?

Reply:
Yes, but I think overall this just allows people to read the paper directly from the browser instead of needing to download it first as a pdf. It also seems like this is optimized for mobile ? I can confirm that it looks much better on mobile.

Reply:
Does mobile browser can change the CSS?
Adding the `ltx_align_left` after `ltx_lst_language_c` or removing `ltx_align_center` in `<figure id="LST1"...>` looks pretty good on computer

Reply:
I am not sure what you mean. Where do you want to add the `ltx_align_left` ?

Reply:
![image](https://user-images.githubusercontent.com/19565938/112273709-9c5ace80-8cb8-11eb-9fd1-ec1dc7d04619.png)
The above figure is the result when doing the previous commit's change.
I would like to know whether it has bad impact on mobile side. (Could mobile browser modify the css?)
Moreover, we can create an issue on arxiv-vanity.

Reply:
Ah, okay. Yeah, I am not sure if mobile browsers have inspect element or something similar. But I think it is a good idea to create an issue on arxiv vanity.